### PR TITLE
Added section on cookies and routes

### PR DESCRIPTION
**Pull request by bfallonf**
For [trello card](https://trello.com/c/aeJZjuVv/680-document-allow-routes-to-set-the-cookie-names-for-session-stickiness-ingress).

Will ask for review over email.

For 3.7 only.

cc @pecameron

### CODE REVIEW

**Review by knobunc — APPROVED** [review]
LGTM.  Thanks @bfallonf

**Comment by bfallonf** [comment]
Great. Thanks @knobunc . @openshift/team-documentation peer review plz

**Comment by bfallonf** [comment]
[rev_history]
|xref:../dev_guide/routes.adoc#dev-guide-routes[Routes]
|Added the xref:../dev_guide/routes.adoc#dev-guide-routes-allowing-endpoints-to-control-cookies[Allowing Route Endpoints to Control Cookie Names] section.
%

File: dev_guide/routes.adoc [hunk 1]
```diff
@@ -270,6 +270,37 @@ xref:../architecture/networking/routes.adoc#path-based-routes[path-based
...
 
+[[dev-guide-routes-allowing-endpoints-to-control-cookies]]
+== Allowing Route Endpoints to Control Cookie Names
+
+{product-title} provides sticky sessions, which enables stateful application
+traffic by ensuring all traffic hits the same endpoint. However, if the endpoint
+pod terminates, whether through restart, scaling, or a change in configuration,
+this statefulness can disappear.
+
+{product-title} can use cookies to configure session persistence. The router selects an endpoint to handle any user requests, and creates a cookie for the session. The cookie is passed back in the response to the request and the user sends the cookie back with the next request in the session. The cookie tells the router which endpoint is handling the session, ensuring that client requests use the cookie so that they are routed to the same pod.
```
Comment (gaurav-nelson) [inline dev_guide/routes.adoc]: Would this be better?
s/{product-title} can use cookies/{product-title} uses cookies

**Comment by gaurav-nelson** [comment]
@bfallonf LGTM  :bird:

**Comment by bfallonf** [comment]
Thanks @gaurav-nelson . I'll merge.

**Pull request merged**